next.config.js

```diff
@@ -12,7 +12,11 @@ const moduleExports = {
     // Warning: This allows production builds to successfully complete even if
     // your project has ESLint errors.
     ignoreDuringBuilds: false,
-  }
+  },
+  env: {
+    MATOMO_URL: process.env.MATOMO_URL,
+    MATOMO_SITE_ID: process.env.MATOMO_SITE_ID,
```

Comment:
This looks redundant, I don't think this adds anything at all.

Also if these are supposed to be visible to the browser then "NEXT_PUBLIC_" should be prefixed: https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

Reply:
if I didn't add them here, for some reason I haven't seen the env vars I'm not sure how they are parsed?